admin/includes/modules/collect_info.php

```diff
@@ -438,12 +442,43 @@
       ?>
     </div>
   </div>
+  <div class="well" style="color: #31708f;background-color: #d9edf7;border-color: #bce8f1;padding: 10px 10px 0 10px;">
```

Comment:
I'm a bit surprised to see all this hard-coded styling...

Reply:
> I'm a bit surprised to see all this hard-coded styling...

For consistency, it was cloned from similar `well` styling implemented earlier in the same page.
A PR is welcome to adapt both to use whatever custom Bootstrap theme is active.

Reply:
maybe best left until after https://github.com/zencart/zencart/pull/6173

Reply:
Done in PR #6244